HQSmokeTests/testPages/users/org_structure_page.py

```diff
@@ -196,3 +205,24 @@ def cleanup_location(self):
         self.click(self.delete_org_level)
         self.wait_to_click(self.save_btn_delete)
         print("Org level deleted successfully")
+
+    def archive_location(self):
+        self.wait_to_click(self.org_menu_link_text)
+        active_loc = self.get_text(self.archive_this_loc)
+        self.wait_to_click(self.archive_buttton)
+        self.wait_to_click(self.archive_button_popup)
+        self.is_present_and_displayed(self.archive_success_message)
+        self.wait_to_click(self.show_arhcived_locations)
+        self.is_present_and_displayed(self.archive_this_loc)
+        archived_loc = self.get_text(self.archive_this_loc)
+        assert archived_loc == active_loc, "Location is not Archived"
```

Comment:
What are we checking here, the text of the location archived shouldn't be changed after being archived?

It's a good check to add, but then we should be marking this test case as pass if:
1. the location is present on the archived page
 This is checked here I believe ` self.is_present_and_displayed(self.archive_this_loc)`
2. NOT present on the landing page2. 
But do we check this?